Installation/installation.json

```diff
@@ -83,15 +83,6 @@
                 }
               }
             }
-          },
-          {
-            "https://vng.opencatalogi.nl/EntityEndpoint/ZaakInformatieObject.endpoint.json": {
-              "out": {
-                "body": {
-                  "mapping": "https://development.zaaksysteem.nl/mapping/xxllnc.XxllncZaakInformatieobjectResultOutMapping.mapping.json"
-                }
-              }
-            }
```

Comment:
Could you reintroduce these lines?

Reply:
This mapping does not exist in the bundle..

Reply:
Ah, I forgot to rename it earlier, here's the correct $id:

`"$id": "https://development.zaaksysteem.nl/mapping/xxllnc.XxllncZaakInformatieobjectBodyOutMapping.mapping.json",`